code/_globalvars/genetics.dm

```diff
@@ -4,7 +4,7 @@ var/COLOURBLINDBLOCK = 0
 var/DEAFBLOCK = 0
 var/HULKBLOCK = 0
 var/TELEBLOCK = 0
-var/FIREBLOCK = 0
+var/COLDRESBLOCK = 0
```

Comment:
Ok. Here's why this is renamed.

Because the block for cold resistance is named fireblock, and the block for heat resistance is named coldblock.

Confusing as heck, right?

Reply:
I agree